frontend/src/metabase/dashboard/components/Dashboard/Dashboard.styled.tsx

```diff
@@ -140,6 +140,8 @@ export const ParametersAndCardsContainer = styled.div<{
     overflow-x: clip;
   }
   padding-bottom: 40px;
+  /* Makes sure it doesn't use all the height, so the actual content height could be used in embedding #37437 */
+  align-self: flex-start;
```

Comment:
![image](https://github.com/metabase/metabase/assets/1937582/439997a7-e5c2-40c0-9471-c3637ae75b41)

Previously this flex element stretched its height to fit the container, which makes it harder to get the total dashboard height since we need to sum more sub-elements.

By doing this we don't need to sum dashboard parameter height and card container heights. We can use this element height directly.